.github/workflows/ci.yml

```diff
@@ -361,6 +361,7 @@ jobs:
             -e REDIS_HOST=sentry_redis \
             -e REDIS_PORT=6379 \
             -e REDIS_DB=1 \
+            -e ENABLE_AUTORUN_MIGRATION_SEARCH_ISSUES=1 \
```

Comment:
Why this change?

Reply:
It was breaking CI to not have it. It's in master now